### ISSUE TITLE
Dist shouldn't be checked in to source control

### ISSUE THREAD

**Issue by Daniel15** [stdlib-js/stdlib#155]
The `dist` directory is a build artifact. It should not be checked in to source control and should instead be added to `.gitignore`. It's too easy for dist to fall out of sync when it's in source control, and it makes people significantly more likely to hit merge conflicts.

The `dist` directory should be included as part of the npm package (which allows usage via Unpkg, and for people to download it), but should not be in the Git repo.

**Comment by kgryte** [comment]
@Daniel15 Thanks for comments. However, the inclusion of the `dist` directory is intentional. We have automated processes for ensuring distributable files are updated whenever we publish a new package version. While npm is the dominant package registry, it is not the only means for consuming packages; I, and others, have frequently needed to consume the project independently of the official npm releases. We do so by installing from GitHub. Lastly, committing to source control and hosting on GitHub allows us to freely host static files which individuals can copy to their projects without needing to explicitly install anything.

In short, we acknowledge that some people will disagree with our decision, but we believe the benefits outweigh the costs.

**Comment by Daniel15** [comment]
You can load static files from npm packages using unpkg though.

Wouldn't it make more sense to use the "releases" section of Github?

**Comment by kgryte** [comment]
Re: npm pkgs. No argument there. The point is that we also want a means for consuming distributable files independent of npm. Meaning, we don't want to create a special npm release in order to consume distributable files.

Re: releases. Not interested in tucking away the distributable files. In this case, we prefer them being more readily accessible on, say, GitHub, and immediately available upon cloning the repository. For instance, these bundles are used when launching the browser REPL which is commonly used during development. As the bundling process can consume time and resources, we want to avoid introducing this overhead for particular development workflows.
